crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__PLR1714_repeated_equality_comparison.py.snap

```diff
@@ -182,12 +182,12 @@ repeated_equality_comparison.py:18:1: PLR1714 [*] Consider merging multiple comp
 16 16 | "a" == foo or foo == "b" or "c" == foo
 17 17 | 
 18    |-foo == bar or baz == foo or qux == foo
-   18 |+foo in (bar, baz, qux)
+   18 |+foo in (bar, baz) or qux == foo
```

Comment:
This seems incorrect? Or, is this expected? Other snapshot updates are also similar.

Reply:
Oops, thanks.